fastlane/Fastfile

```diff
@@ -43,4 +36,21 @@ platform :ios do
 
   end
 
+  desc "Generate docs"
+  lane :generate_docs do
+    sh("jazzy --output='/Users/distiller/amplify-ios/docs' --min-acl=internal -x -workspace,/Users/distiller/amplify-ios/Amplify.xcworkspace,-scheme,Amplify")
+    commit_push()
+  end
+
+  desc "Commit and push"
+  private_lane :commit_push do
```

Comment:
Can you explain the branch strategy here? I see we're pushing to gh-pages below on line 53, but I'm unclear whether we're expected to be on gh-pages to begin with, or whether this is expected to run on master. If on master, does that mean there exists the possibility of merging the `docs` directory to `master` if we don't clean up the branch `--set-upstream` before exiting this lane?